### PR TITLE
Rats cicd

### PR DESCRIPTION
**Pull request by YangLiang3**

### CODE REVIEW

File: .github/workflows/pr_rats_tls.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        docker exec $inclavare_test bash -c 'sleep 10 && ${{ env.RATS_TLS_BINDIR }}/rats-tls-client -l debug -a nullattester -v nullverifier -t openssl -c openssl -m -p ${{ steps.random-port-generator3.outputs.random-port }}'
+
+    - name: Clean up the github workspace
+      if: ${{ always() }}
+      run:
+        docker restart $inclavare_test;
+        docker exec $inclavare_test bash -c 'rm -rf /root/inclavare-containers/${{ matrix.tag }}'
+
+    - name: Kill the test container
+      run: docker stop $inclavare_test
```
Comment (haosanzi) [inline .github/workflows/pr_rats_tls.yml:116]: Please do NOT delete `tools/docker/Dockerfile-inclavare-dev-centos8.2` and `tools/docker/Dockerfile-inclavare-dev-ubuntu18.04` in this pr

File: .github/workflows/nightly-centos-sgx1.yml [hunk 2]
```diff
@@ -97,6 +97,7 @@ jobs:
         sed -i 's/shelter//g' Makefile
         sed -i 's/inclavared//g' Makefile
         sed -i 's/enclave-tls//g' Makefile
+        sed -i 's/rats-tls//g' Makefile
```
Comment (haosanzi) [inline .github/workflows/nightly-centos-sgx1.yml:100]: There is no need to `sed -i 's/rats-tls//g' Makefile` since there is no `rats-tls` in Makefile

**Comment by haosanzi** [comment]
Please fix the compile error in Basic Compilation Check(alinux2)

File: .github/workflows/pr_basic_compilation_check.yml [hunk 3]
```diff
@@ -35,6 +35,14 @@ jobs:
           source /root/.bashrc && make && make install && make clean &&
             make SGX=1 && make clean && make OCCLUM=1
 
+      - name: Compile "rats-tls"
+        if: ${{ !contains(matrix.sgx, 'alinux2') }}
```
Comment (haosanzi) [inline .github/workflows/pr_basic_compilation_check.yml:39]: Please change `if: ${{ !contains(matrix.sgx, 'alinux2') }}` to `if: ${{ !contains(matrix.tag, 'alinux2') }}`

**Comment by haosanzi** [comment]
Please add LVI binutils tools in [compile test dockerfile](https://github.com/alibaba/inclavare-containers/blob/master/.github/workflows/docker/Dockerfile-compilation-testing-centos8.2) and update the images, otherwise the [compiler check](https://github.com/alibaba/inclavare-containers/pull/1368/checks?check_run_id=3541445201) will error.

**Comment by YangLiang3** [comment]
> Please add LVI binutils tools in [compile test dockerfile](https://github.com/alibaba/inclavare-containers/blob/master/.github/workflows/docker/Dockerfile-compilation-testing-centos8.2) and update the images, otherwise the [compiler check](https://github.com/alibaba/inclavare-containers/pull/1368/checks?check_run_id=3541445201) will error.

It has been updated yesterday

**Comment by haosanzi** [comment]
Will merge this pr after passing the rats-tls test in the private repo

**Pull request merged**